tests/test_query_features.py

```diff
@@ -427,3 +427,25 @@ def test_entity_gaz_query_features(
         else:
             assert expected_value == extracted_features[feature_key]
     entity_recognizer.fit()
+
+
+@pytest.mark.parametrize(
+    "query, expected_entity_type, domain, intent",
+    [
+        # system entity should be detected
+        ('2am', 'sys_time', 'times_and_dates', 'change_alarm'),
+        ('4:30 pm', 'sys_time', 'times_and_dates', 'change_alarm'),
+        ('tomorrow at noon', 'sys_time', 'times_and_dates', 'change_alarm'),
+        # custom entity in training data should be detected
+        ('bathroom', 'location', 'smart_home', 'set_thermostat'),
+        ('bedroom', 'location', 'smart_home', 'set_thermostat'),
+        ('balcony', 'location', 'smart_home', 'set_thermostat'),
+        # gaz entity should be detected even if not in training data
+        ('powder room', 'location', 'smart_home', 'set_thermostat'),
+        ('living room', 'location', 'smart_home', 'set_thermostat'),
```

Comment:
Were all of these tests failing previously?

Reply:
only the `# system entity should be detected` tests cases:
```
1. `2am`
2. `4:30pm`
```

I added the other tests cases to make sure the other featurizers like `in-gaz-span-seq` work properly as well.